### ISSUE TITLE
Insert page fail when final PDF is >= 20 pages 

### ISSUE THREAD

**Issue by Yu-AnChen** [chunyenHuang/hummusRecipe#12]
```javascript
const finalPdf = new HummusRecipe(pdf, outputPdf);
    let repeat = 1;
    while (repeat < repeats + 1) {
        finalPdf.insertPage(repeat, otherPdf, 1);
        repeat += 1;
    }
finalPdf.endPDF();
```

I'm not sure where the issue comes from, when `repeats > 9`, I get this `TypeError: unable to append page, make sure it's fine` error.

The primary pdf is an 11-page pdf file, I was trying to insert one identical page after each page in the primary pdf file.

**Comment by chunyenHuang** [comment]
This is fixed in `1.3.5` [commit](https://github.com/chunyenHuang/hummusRecipe/commit/a4aeabc7fb52ee4fb60719f6bc607024e9fa1deb)

**Comment by Yu-AnChen** [comment]
Awesome! Thanks!
